Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -35,7 +35,7 @@
 --someAny disabled # opaqueGenericParameters
 --elseposition same-line #elseOnSameLine
 --guardelse next-line #elseOnSameLine
---oneLineForEach wrap #preferForLoop
+--onelineforeach convert #preferForLoop
```

Comment:
Nick updated how the preferForLoop rule works, so I updated our configuration to preserve the behavior from previous versions https://github.com/nicklockwood/SwiftFormat/pull/1490#issuecomment-1882731976